### PR TITLE
Fixes #30664 - add installable models for host

### PR DESCRIPTION
**Pull request by snagoor**
added new `installable_errata` and `installable_rpms` models for Report Templates that needs purely `installable` report data.

### CODE REVIEW

**Comment by ares** [comment]
This needs a rebase now

**Comment by snagoor** [comment]
> This needs a rebase now

@ares done :+1:

File: app/lib/katello/concerns/base_template_scope_extensions.rb [hunk 1]
```diff
@@ -78,6 +78,18 @@ def host_latest_applicable_rpm_version(host, package)
         host.applicable_rpms.where(name: package).order(:version_sortable).limit(1).pluck(:nvra).first
       end
 
+      def host_installable_errata_ids(host)
+        host.installable_errata.map(&:errata_id)
```
Comment (jlsherrill) [inline app/lib/katello/concerns/base_template_scope_extensions.rb:82]: using  .pluck()  would be far more efficient

Comment (jlsherrill) [reply app/lib/katello/concerns/base_template_scope_extensions.rb:82]:
```
         host.installable_errata.pluck(&:errata_id)
```

File: app/models/katello/host/content_facet.rb [hunk 2]
```diff
@@ -15,12 +15,14 @@ class ContentFacet < Katello::Model
 
       has_many :content_facet_errata, :class_name => "Katello::ContentFacetErratum", :dependent => :delete_all, :inverse_of => :content_facet
       has_many :applicable_errata, :through => :content_facet_errata, :class_name => "Katello::Erratum", :source => :erratum
+      has_many :installable_errata, :through => :content_facet_errata, :class_name => "Katello::Erratum", :source => :erratum
```
Comment (jlsherrill) [inline app/models/katello/host/content_facet.rb:18]: I don't think this makes sense, as the errata in this association are 'applicable', but not necessarily installable.  You'd need to call the installable_errata' method below to calculate that.  Same comment for the installable_rpms relationship down below

**Comment by ezr-ondrej** [comment]
@snagoor can you take a look and rebase, please?

**Pull request closed without merging**